content/10.methods.md

```diff
@@ -43,23 +43,41 @@ For other keynote speakers, we gleaned their full name as well as the year in wh
-The PM set contained 12963 manuscripts published in Bioinformatics, 6349 published in PLOS Computational Biology, and 9124 published in BMC Bioinformatics.
-Despite being more comprehensive, the PM set of manuscripts did not provide corresponding author information.
-We inferred that the leading senior author of a manuscript would be in the last author position.
-We selected the last author from each paper as our set for consideration.
-
-PMC provided a more limited resource but contained more detailed corresponding author information.
-The PMC set contained 4797 articles published in Bioinformatics, 6296 published in PLOS Computational Biology, and 9099 published in BMC Bioinformatics.
-We examined the corresponding author number for each paper and found that all but 359 had at least one corresponding author.
-We manually examined a subset of the manuscripts without corresponding authors and found that many were editorials (e.g., the announcement of PLOS Computational Biology).
-The vast majority of publications (17064) had only a single corresponding author.
-We used the corresponding authors from the PMC set as our set for consideration.
+### Corresponding author extraction
+
+We assumed that research advisors in the field would be those most likely to be invited for keynotes or to be honored as Fellows.
+Therefore, we collected corresponding author names to assess the composition of the field, weighted by research output as judged by publication.
+
+We evaluated two resources for extracting corresponding authors from papers: [PubMed](https://pubmed.ncbi.nlm.nih.gov/) and [PubMed Central](https://www.ncbi.nlm.nih.gov/pmc/) (PMC).
+Both resources are provided by the U.S. National Library of Medicine and index scholarly articles.
+PubMed contains a record for every article published in journals it indexes (30 million records total circa 2020) and provides abstracts but not fulltext.
+PMC, which provides fulltext access, does not contain every article from every journal (5.9 million records total circa 2020).
+In general, open access journals will deposit their entire catalog to PMC (e.g. _BMC Bioinformatics_ & _PLOS Computational Biology_), while toll access journals (e.g. _Bioinformatics_) will only deposit articles when funders require it.
+Since PMC requires publishers to submit fulltext articles in a structured XML format, the machine-readability and breadth of metadata in PMC is often superior to PubMed.
+
+Of PMC's 5.9 million fulltext articles, only 2.7 million are part of the "[Open Access Subset](https://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/)" which allows for downloading the structured fulltext as opposed to just viewing the article online.
+Luckily however, since we only cared about authorship information, we were able to download structured frontmatter (rather than fulltext) records from PMC's [OAI-PMH service](https://www.ncbi.nlm.nih.gov/pmc/tools/oai/), such that we were not limited to just the OA Subset.
```

Comment:
```suggestion
However, authorship information does not require full text records.
We were able to download structured frontmatter (rather than fulltext) records from PMC's [OAI-PMH service](https://www.ncbi.nlm.nih.gov/pmc/tools/oai/), so we were not limited to just the Open Access Subset.
```